includes/apple-exporter/builders/class-text-styles.php

```diff
@@ -0,0 +1,123 @@
+<?php
```

Comment:
There is an open issue for moving everything to WP standards, so this is on the roadmap. I'm trying to refactor as I go, but if I'm only applying minor changes to a file, I won't refactor the entire thing.